.github/workflows/build.yml

```diff
@@ -55,13 +51,20 @@ jobs:
           distribution: 'temurin'
           cache: 'maven'
```

Comment:
some drawbacks have been seen with the cache system in setup-java, so using `actions/cache` is now preferred (AAE-10961)

Reply:
Fixed